### ISSUE TITLE
New Xiangqi Board theme

### ISSUE THREAD

**Issue by tititogepi** [Mind-Sports-Games/lila#593]
<img width="347" alt="image" src="https://github.com/Mind-Sports-Games/lila/assets/129761185/3c80a944-256c-4fbd-8c38-90cdc8d1f2ec">
Hello!
Can Playstrategy add this boards theme? It is very nice and it already exist on pychess!
:)
tititogepi

**Comment by JamesHeppell** [comment]
That board does have a nice texture to it. It would be nice to pair it with a mini-xiangqi texture board which I don't think is available on pychess? Our board images are currently set across our game families (e.g. Xiangqi for both itself and mini Xiangqi), therefore we would want to add a pair of images if possible.

**Comment by tititogepi** [comment]
Yes this board isn't available for mini xiangqi on pychess. Yes it'd be great! :)
